scripts/reboot

```diff
@@ -2,12 +2,9 @@
 
 function stop_sonic_services()
 {
-    echo "Stopping sonic services..."
-    systemctl stop swss
-    systemctl stop teamd
-    systemctl stop bgp
-    systemctl stop lldp
-    systemctl stop snmp
+    echo "Stopping syncd..."
+    docker exec -it syncd /usr/bin/syncd_request_shutdown --cold > /dev/null
+    sleep 3
```

Comment:
does this work on Mellanox platform?

Reply:
It works on all platforms. Fast-reboot is also calling this to terminate syncd.

Regards,
Ying